CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 TBD
 
+## [v2.0.2](https://github.com/medibloc/panacea-core/releases/tag/v2.0.2) - 2022-03-08
```

Comment:
Is there a reason why the same version was added?

Reply:
Hmm.. It's may be wrong commit. I'll delete it.